docs_src/OVMS/quick_stream_density.md

```diff
@@ -73,6 +73,26 @@ There are several pipeline profiles to choose from for running pipeline stream d
         The benchmark.sh script automatically cleans all running Docker containers after it is done.
 
 
+    If the hardware supports, then one can also run the benchmarking on different devices like CPU or GPU.  This can be done through the environment variable `DEVICE`.  The following is an example to run the object_detection profile using GPU:
+
+    ```bash
+    PIPELINE_PROFILE="object_detection" RENDER_MODE=0 DEVICE="GPU.0" sudo -E ./benchmark.sh --stream_density 14.95 --logdir object_detection/data --duration 120 --init_duration 40 --platform dgpu.0 --inputsrc rtsp://127.0.0.1:8554/camera_1
+    ```
+
+    !!! Note
+        The performance of running object detection benchmarking should be better while running on GPU using model precision FP32.  If supported, then you can change the model precision by going to the folder `configs/opencv-ovms/models/2022` from the root of project folder and editing the `base_path` for that particular model in the `config_template.json` file.  For example, you can change the the base_path of `FP32` to `FP16` assuming the precision `FP16` of the model is available: 
```

Comment:
"assuming the precision `FP16` of the model is available", would it be better if we provide directory location where user can see/put FP16/FP32 model files?

Reply:
yea, i can add some directory structure in this section: but user don't put FP16/32 model files themselves, it is done through model downloader as part of process- as long as it is supported by OpenVINO downloader and it is done through updating the config_templage.json as part of process as of today.

Reply:
I was thinking to make it easy and no guessing to the user, if user can find a correct FP16 model files and put them in FP16 folder, it would still work as of today, right? if it is, I think providing directory of where FP16 model files would be helpful for user.

Reply:
no, don't suggest something off the process, it could bring more troubles or issues than its worth

Reply:
if you see a good approach you can create an issue and work on it. this just tries to fix the current issue based on the current process we have @vli11